### PR TITLE
refactor: removed synthetics & kotlin android extension

### PR DESCRIPTION
**Pull request by PratyushSingh07**
- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: compose/build/intermediates/incremental/debug/packageDebugResources/merger.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...ile\compose\build\generated\res\resValues\debug"/></dataSet><dataSet aapt-namespace="http://schemas.android.com/apk/res-auto" config="main" generated-set="main$Generated" ignore_pattern="!.svn:!.git:!.ds_store:!*.scc:.*:&lt;dir>_*:!CVS:!thumbs.db:!picasa.ini:!*~"><source path="C:\Users\aries\AndroidStudioProjects\mifos-mobile\compose\src\main\res"/><source path="C:\Users\aries\AndroidStudioProjects\mifos-mobile\compose\build\generated\res\rs\debug"/><source path="C:\Users\aries\AndroidStudioProjects\mifos-mobile\compose\build\generated\res\resValues\debug"/></dataSet><dataSet aapt-namespace="http://schemas.android.com/apk/res-auto" config="debug$Generated" generated="true" ignore_pattern="!.svn:!.git:!.ds_store:!*.scc:.*:&lt;dir>_*:!CVS:!thumbs.db:!picasa.ini:!*~"><source path="C:\Users\aries\AndroidStudioProjects\mifos-mobile\compose\src\debug\res"/></dataSet><dataSet aapt-namespace="http://schemas.android.com/apk/res-auto" config="debug" generated-set="debug$Generated" ignore_pattern="!.svn:!.git:!.ds_store:!*.scc:.*:&lt;dir>_*:!CVS:!thumbs.db:!picasa.ini:!*~"><source path="C:\Users\aries\AndroidStudioProjects\mifos-mobile\compose\src\debug\res"/></dataSet><mergedItems/></merger>
```
Comment (jawidMuhammadi) [inline compose/build/intermediates/incremental/debug/packageDebugResources/merger.xml:2]: What is this file? Isn't supposed to be on git ignore?

File: compose/build/intermediates/incremental/mergeDebugShaders/merger.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merger version="3"><dataSet config="main" ignore_pattern="!.svn:!.git:!.ds_store:!*.scc:.*:&lt;dir>_*:!CVS:!thumbs.db:!picasa.ini:!*~"><source path="C:\Users\aries\AndroidStudioProjects\mifos-mobile\compose\src\main\shaders"/></dataSet><dataSet config="debug" ignore_pattern="!.svn:!.git:!.ds_store:!*.scc:.*:&lt;dir>_*:!CVS:!thumbs.db:!picasa.ini:!*~"><source path="C:\Users\aries\AndroidStudioProjects\mifos-mobile\compose\src\debug\shaders"/></dataSet></merger>
```
Comment (jawidMuhammadi) [inline compose/build/intermediates/incremental/mergeDebugShaders/merger.xml:2]: Same here?

**Review by jawidMuhammadi — CHANGES_REQUESTED** [review]

File: compose/build/intermediates/incremental/mergeDebugShaders/merger.xml (hunk 2 above)
Comment (PratyushSingh07) [reply compose/build/intermediates/incremental/mergeDebugShaders/merger.xml:2]: I tried to stash this compose module entirely before pushing the code @jawidMuhammadi . I am not really sure why it didnt work

**Comment by PratyushSingh07** [comment]
@jawidMuhammadi , I have now created a .gitgnore file inside of compose module and added /build to it 
image

Will  this work ?

**Comment by PratyushSingh07** [comment]
Nvm @jawidMuhammadi , I am closing this and opening a new PR instead. I will stash cmopose module entirely and then open a draft PR with the changes that you had asked

**Pull request closed without merging**